pyproject.toml

```diff
@@ -40,7 +40,7 @@ Unidecode = ">=1.3.2"
 python-dotenv = ">=0.19.0"
 frictionless = {extras = ["pandas"], version = ">=4.40.8"}
 thefuzz = ">=0.19.0"
-python-Levenshtein = ">=0.12.2"
+python-Levenshtein = "^0.12.2"
```

Comment:
Could be useful to add a note explaining why is it pinned and link the issue. This will make it easier to unpin in the future.

Reply:
Good point!